enterprise/internal/campaigns/reconciler/executor.go

```diff
@@ -559,25 +559,22 @@ func buildPushConfig(extSvcType, cloneURL string, a auth.Authenticator) (*protoc
 	}
 
 	switch av := a.(type) {
+	case *auth.OAuthBearerTokenWithSSH:
+		if err := setOAuthTokenAuth(u, extSvcType, av.Token); err != nil {
+			return nil, err
+		}
```

Comment:
I think you can do
```suggestion
	case *auth.OAuthBearerToken, *auth.OAuthBearerTokenWithSSH:
		if err := setOAuthTokenAuth(u, extSvcType, av.Token); err != nil {
			return nil, err
		}
```

and remove the other branch.

Reply:
Hmm I think when two types are specified, the type of `av` in here will still be the interface they have in common, which is `auth.Authenticator`, and since that interface doesn't have a field `Token`, I think it wouldn't work. 😞  At least, that's what I get from the compiler output.

Reply:
Yep. I had this happen a bunch with GitHub and GitLab metadata, which are often similar enough that the symbols are the same, but it doesn't work in Go. :cry: